specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/machineLearningServices.json

```diff
@@ -4851,6 +4856,24 @@
       ],
       "type": "object"
     },
+    "ImageMetadata": {
+      "type": "object",
+      "description": "Specifies the Image Metadata about current ComputeInstance",
+      "properties": {
+        "currentImageVersion": {
+          "type": "string",
+          "description": "Indicates the current version of OS Base Image, Compute Instance is running on."
+        },
+        "latestImageVersion": {
+          "type": "string",
+          "description": "Indicates the latest OS Base Image available for the Compute Instance."
+        },
+        "isLatestOsImageVersion": {
+          "type": "boolean",
+          "description": "Indicates whether the Compute Instance is running latest OS Base Image or not."
```

Comment:
Specifies whether this compute instance is running on the latest operating system image.